libr/core/cmd_debug.c

```diff
@@ -5118,12 +5118,12 @@ static int cmd_debug(void *data, const char *input) {
 		case ' ': // "dd"
 			// TODO: handle read, readwrite, append
 			{
-				RBuffer *buf = r_core_syscallf (core, "open", "%s, %d, %d", input + 2, 2, 0644);
+				RBuffer *buf = r_core_syscallf (core, "open", "\"%s\", %d, %d", input + 2, 2, 0644);
```

Comment:
Maybe good to escape the string to avoid quotes and other problematic chars before passing it.

also if the user wants to pass a numeric value with the address of the allocated string that wont work. So maybe we can also check if the passed string is a number and use pfmt64x

what do you think?